backend/tests/test_accounts.py

```diff
@@ -0,0 +1,185 @@
+import pytest
+
```

Comment:
nit: your tests are all well organized, thus you can avoid parameterizing the `expected_status_code` since in each test it is the same value.